utilities/io500_fixed.sh

```diff
@@ -5,29 +5,71 @@
 
 set -euo pipefail   # give bash better error handling.
 export LC_NUMERIC=C  # prevents printf errors
+export LC_ALL=C
+IO500_MIN_WRITE_RUNTIME=300
 
 function main {
+  set_defaults
   check_variables
   output_description
   core_setup
+  clean_cache
   ior_easy "write"
+  clean_cache
   mdt_easy "write"
+  clean_cache
   touch $timestamp_file  # this file is used subsequently by the find command
   ior_hard "write"
+  clean_cache
   mdt_hard "write"
+  clean_cache
   myfind
   ior_easy "read"
+  clean_cache
   mdt_easy "stat"
+  clean_cache
   ior_hard "read"
+  clean_cache
   mdt_hard "stat"
+  clean_cache
   mdt_easy "delete"
+  clean_cache
   mdt_hard "read"
+  clean_cache
   mdt_hard "delete"
   mdreal
   cleanup
   output_score
 }
 
+function set_defaults {
+  # Set default values, they may be overwritten by the user if necessary
+  io500_mdtest_easy_files_per_proc=${io500_mdtest_easy_files_per_proc:-250000}
+  io500_mdtest_hard_files_per_proc=${io500_mdtest_hard_files_per_proc:-500000}
```

Comment:
Can you please set this to 950000?  950000 is ten times the highest value that has ever been set.